### PR TITLE
Update action to block and increase priority for allow

### PR DESCRIPTION
**Pull request by aegal**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: cns/NetworkContainerContract.go [hunk 1]
```diff
@@ -244,6 +244,16 @@ type ValidAclPolicySetting struct {
 	Priority        uint16 `json:","`
 }
 
+const (
+	ActionTypeAllow string = "Allow"
+
```
Comment (tamilmani1989) [inline cns/NetworkContainerContract.go:249]: remove space

File: cns/NetworkContainerContract.go [hunk 2]
```diff
@@ -244,6 +244,16 @@ type ValidAclPolicySetting struct {
...
+	ActionTypeBlock string = "Block"
+
```
Comment (tamilmani1989) [inline cns/NetworkContainerContract.go:251]: no space between each constants

File: cns/hnsclient/hnsclient_windows.go [hunk 3]
```diff
@@ -67,6 +67,9 @@ const (
 	// aclPriority200 indicates the ACL priority of 200
 	aclPriority200 = 200
 
+	// aclPriority1000 indicates the ACL priority of 1000
+	aclPriority1000 = 1000
```
Comment (tamilmani1989) [inline cns/hnsclient/hnsclient_windows.go:71]: why choose 1000? does this signify higher or lower priority

Comment (aegal) [reply cns/hnsclient/hnsclient_windows.go:71]: Higher priority for allows, right now we block at 2000 and allow at 200. It allows for the orchestrator to specify a rule that is block (example: 999 block all below) and allow rules to exclude each port/protocol/src/dest address.

File: cns/NetworkContainerContract.go (hunk 2 above)
Comment (aegal) [reply cns/NetworkContainerContract.go]: changed.

**Pull request merged**